LEAF_Request_Portal/admin/Group.php

```diff
@@ -94,6 +102,11 @@ public function addMember($member, $groupIDs)
                               ':groupID' => (int)$group, );
                 $res = $this->db->prepared_query('INSERT INTO users (userID, groupID)
                                                     VALUES (:userID, :groupID)', $vars);
+                
+                $this->dataActionLogger->logAction(\DataActions::ADD,\LoggableTypes::EMPLOYEE,[
+                    new LogItem("users","userID", $member, $this->getEmployeeDisplay($member)),
```

Comment:
nit-pick: Should there be inconsistencies with the use of ```/LogItem``` usage here vs in ```LEAF_Nexus/sources/Group.php```

Reply:
Updated to match.